src/v2/Components/ArtistSeriesRail/ArtistSeriesItem.tsx

```diff
@@ -26,10 +26,10 @@ export const ArtistSeriesItem: React.SFC<Props> = props => {
           width={160}
           style={{ objectFit: "cover", objectPosition: "center" }}
         />
-        <Box mt={1} ml={1}>
+        <Box my={1} mx="15px">
```

Comment:
Why `15px`?

Reply:
Fixed! (Was in the spec but confirmed w/ Nicole that we can snap to `20` here).